zone/tasks.cpp

```diff
@@ -280,7 +280,138 @@ bool TaskManager::LoadTasks(int singleTask)
 
 	return true;
 }
+std::string TaskManager::SaveClientStateQuery(Client *c, ClientTaskState *state)
```

Comment:
Ignoring the fact VS lets this compile for some reason. This is also written in a way that may possibly fail NRVO and since this is a performance focused commit, that is a very relevant point :P

Reply:
hopefully this is resolved with pass by ref